app/app/settings.py

```diff
@@ -29,7 +29,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['localhost']
```

Comment:
This is duplicated below.

Reply:
yeah let's just keep `ALLOWED_HOSTS = ['localhost']`